src/components/ftbot/TradeListNav.vue

```diff
@@ -9,26 +9,42 @@
         >Trade Navigation {{ sortNewestFirst ? '&#8595;' : '&#8593;' }}
       </b-list-group-item>
       <b-list-group-item
-        v-for="trade in sortedTrades"
+        v-for="(trade, i) in sortedTrades"
         :key="trade.open_timestamp"
         button
         class="d-flex flex-wrap justify-content-between align-items-center py-1"
         :title="`${trade.pair}`"
         :active="trade.open_timestamp === selectedTrade.open_timestamp"
         @click="onTradeSelect(trade)"
       >
-        <div>
-          <span v-if="botStore.activeBot.botState.trading_mode !== 'spot'">{{
-            trade.is_short ? 'S-' : 'L-'
-          }}</span>
-          <DateTimeTZ :date="trade.open_timestamp" />
+        <div class="d-flex flex-column">
+          <div>
+            <span v-if="botStore.activeBot.botState.trading_mode !== 'spot'">{{
+              trade.is_short ? 'S-' : 'L-'
+            }}</span>
+            <DateTimeTZ :date="trade.open_timestamp" />
+          </div>
+          <TradeProfit :trade="trade" class="my-1" />
+          <ProfitPill
+            v-if="backtestMode"
+            :profit-ratio="trade.profit_ratio"
+            :stake-currency="botStore.activeBot.stakeCurrency"
+          />
         </div>
-        <TradeProfit :trade="trade" />
-        <ProfitPill
-          v-if="backtestMode"
-          :profit-ratio="trade.profit_ratio"
-          :stake-currency="botStore.activeBot.stakeCurrency"
-        />
+        <b-button size="sm" @click="ordersVisible[i] = !ordersVisible[i]"
```

Comment:
I'm yet to find time to try this - but from the picture, i think this button needs a different styling.

either something like secondary-outline - or an "invisible" button - which will only leave the chevron visible unless there's a hover event.